lib/query.js

```diff
@@ -3783,24 +3784,35 @@ Query.prototype.http = function(args, options, internalOptions) {
     return new Promise(function(resolve, reject) {
```

Comment:
Here you need to chain `self.evalute(options, internalOptions)` after `self.evalute(args[0], internalOptions)

Once you have evaluate the url and the option, you can call `request.get`